wsaa.py

```diff
@@ -65,6 +65,10 @@
     from subprocess import Popen, PIPE
     from base64 import b64encode
 
+if "pytest" in sys.modules:
```

Comment:
You could import subprocess and base64 always, please do not put conditions for tests in the main code

Reply:
Got it, I will remove that line. I added that line of code as the tests were failing due to subprocess and base64 not imported at run time due to the try and except statement